src/test/java/org/elasticsearch/index/query/SimpleQueryStringBuilderTest.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.index.query;
+
+import org.apache.lucene.search.BooleanClause;
+import org.apache.lucene.search.BooleanQuery;
+import org.apache.lucene.search.MatchNoDocsQuery;
+import org.apache.lucene.search.Query;
+import org.apache.lucene.search.TermQuery;
+import org.elasticsearch.index.query.SimpleQueryStringBuilder.Operator;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.hamcrest.Matchers.*;
+
+public class SimpleQueryStringBuilderTest extends BaseQueryTestCase<SimpleQueryStringBuilder> {
+
+    @Override
+    protected SimpleQueryStringBuilder createTestQueryBuilder() {
+        SimpleQueryStringBuilder qb = new SimpleQueryStringBuilder(randomAsciiOfLength(8));
+        if (randomBoolean()) qb.queryName(randomAsciiOfLength(8));
+        if (randomBoolean()) qb.analyzeWildcard(randomBoolean());
+        if (randomBoolean()) qb.lenient(randomBoolean());
+        if (randomBoolean()) qb.lowercaseExpandedTerms(randomBoolean());
+        if (randomBoolean()) qb.locale(randomLocale(getRandom()));
+        if (randomBoolean()) qb.minimumShouldMatch(randomFrom(new String[]{"1", "-1", "75%", "-25%", "2<75%", "2<-25%"}));
+        if (randomBoolean()) qb.analyzer("simple");
+        if (randomBoolean()) qb.defaultOperator(randomFrom(Operator.AND, Operator.OR));
```

Comment:
No need for sorry - will change - I'm also not a huge fan of this way of writing if statements. Especially as I tend to be one of the first to fall into the trap of adding another statement to the statement forgetting to add the brackets.

Left them out here only because when doing the refactoring this formatting seemed to make the repetition just a bit clearer.